src/main/java/org/scijava/io/BytesHandle.java

```diff
@@ -116,6 +116,18 @@ public void seek(final long pos) throws IOException {
 		bytes().position((int) pos);
 	}
 
+	@Override
+	public boolean isReadable() {
+		return true;
+	}
+
+	@Override
+	public boolean isWritable() {
+		// NB: Actually, some ByteBuffers are read-only.
+		// But sadly there is no API for determining this.
```

Comment:
Ah, but we _can_ make a best guess. I suggest:
```
return !(this instanceof HeapByteBufferR || this instanceof DirectByteBufferR)
```
as a heuristic that gets us 99% of the way there. No?